src/plugins/ModActions.ts

```diff
@@ -444,6 +446,14 @@ export class ModActionsPlugin extends ZeppelinPlugin<IModActionsPluginConfig> {
       ),
     );
 
+    const priorWarnAmount = (await casesPlugin.getCaseTypeAmountForUserId(memberToWarn.id, CaseTypes.Warn)) - 1;
+    if (priorWarnAmount >= config.warn_amount_notify_threshhold) {
+      this.sendErrorMessage(
```

Comment:
Perhaps we should ask for verification instead? Kinda like we do above for "Failed to message the user".

Reply:
Sounds like a good idea. Will do on the 13th/14th